lib/db/mysql/patches/patch-021-022.sql

```diff
@@ -0,0 +1,4 @@
+-- Add column to stash the `profileChangedAt` value
+ALTER TABLE codes ADD COLUMN profileChangedAt BIGINT DEFAULT 0;
+
+UPDATE dbMetadata SET value = '22' WHERE name = 'schema-patch-level';
```

Comment:
Is adding a column to the profile server's dbMetadata table a slow process?

Reply:
I don't believe so, but I don't think it will hurt to add `ALGORITHM = INPLACE, LOCK = NONE;` here.

Reply:
Oh, misread thought this was about the alter table above. Updating the `dbMetadata` is a pretty quick operation. FWIW, I still added `ALGORITHM = INPLACE, LOCK = NONE;` to alter.